envoyextensions/extensioncommon/basic_envoy_extender.go

```diff
@@ -112,89 +112,12 @@ func (b *BasicEnvoyExtender) Extend(resources *xdscommon.IndexedResources, confi
 }
```

Comment:
Changes in this file are equivalent to the original PR but slightly different due to refactors that did not require backporting